sqlstore/src/main/resources/db/migration/V8900__hibernatize_everything.sql

```diff
@@ -22,7 +22,25 @@ INSERT INTO ProjectOverview_Sample(projectOverview_overviewId, sample_sampleId)
 DROP TABLE EntityGroup_Elements;
 DROP TABLE EntityGroup;
 
-UPDATE Pool_Elements SET elementType = CASE elementType WHEN 'uk.ac.bbsrc.tgac.miso.core.data.impl.LibraryDilution' THEN 'LDI' WHEN 'uk.ac.bbsrc.tgac.miso.core.data.impl.emPCRDilution' THEN 'EDI' END;
+CREATE TABLE `Pool_Dilution` (
+  `pool_poolId` bigint(20) NOT NULL,
+  `dilution_dilutionId` bigint(20) NOT NULL,
+  PRIMARY KEY (`pool_poolId`,`dilution_dilutionId`),
+  CONSTRAINT `Pool_Dilution_pool_poolId` FOREIGN KEY (`pool_poolId`) REFERENCES `Pool` (`poolId`),
+  CONSTRAINT `Pool_Dilution_dilution_dilutionId` FOREIGN KEY (`dilution_dilutionId`) REFERENCES `LibraryDilution` (`dilutionId`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO Pool_Dilution(pool_poolId, dilution_dilutionId) SELECT
+  pool_poolId, elementId FROM Pool_Elements
+  WHERE elementType = 'uk.ac.bbsrc.tgac.miso.core.data.impl.LibraryDilution';
+INSERT INTO Pool_Dilution(pool_poolId, dilution_dilutionId) SELECT
+  pool_poolId, dilution_dilutionId
+  FROM Pool_Elements JOIN emPCRDilution ON Pool_Elements.elementId = emPCRDilution.dilutionId
+  JOIN emPCR ON emPCR.pcrId = emPCRDilution.emPCR_pcrId
+  WHERE elementType = 'uk.ac.bbsrc.tgac.miso.core.data.impl.emPCRDilution'
+  AND NOT EXISTS (SELECT * FROM Pool_Dilution WHERE Pool_Dilution.pool_poolId = Pool_Elements.pool_poolId AND Pool_Dilution.dilution_dilutionId = emPCR.dilution_dilutionId);
```

Comment:
Aha -- for once, the columns aren't ambiguously-named so they don't need to be aliased. I missed the part where `dilution_dilutionId` came from the emPCR table.